tmt/plugins.py

```diff
@@ -0,0 +1,50 @@
+# coding: utf-8
+
+""" Handle Steps Plugins """
+
+import fmf
+import os
+import importlib
+import pkgutil
+import tmt
+import sys
+
+log = fmf.utils.Logging('tmt').logger
+
+def explore():
+    """ Explore all available plugins """
+
+    # Check all tmt steps for native plugins
+    root = os.path.dirname(os.path.realpath(tmt.__file__))
+    for step in tmt.steps.STEPS:
+        for module in discover(os.path.join(root, 'steps', step)):
+            import_(f'tmt.steps.{step}.{module}')
+
+    # Check environment variable for user plugins
+    try:
+        paths = [
+            os.path.realpath(os.path.expandvars(os.path.expanduser(path)))
+            for path in os.environ['TMT_PLUGINS'].split(' ')]
+    except KeyError:
+        paths = []
```

Comment:
maybe it would be nice to print a debug message, that no `TMT_PLUGINS` was found in environment, so no additional plugins are being loaded?

Reply:
Makes sense. Added a debug message.